docs/src/appendix/sncast/verify.md

```diff
@@ -6,7 +6,13 @@ Required.
 
 The address of the contract that is to be verified.
 
-## `--contract-name <CONTRACT_NAME>`
+
+## `--class-hash, -c <CONTRACT_ADDRESS>`
+Required.
+
+The address of the contract that is to be verified.
```

Comment:
Let's also say that this conflicts with the other argument and exactly one is required.